app/models/saved_recipie.rb

```diff
@@ -0,0 +1,10 @@
+class SavedRecipie < ApplicationRecord
+  ## VALIDATIONS
+    validates :user_id, presence: true
```

Comment:
We might want to validate the assosiation (make sure the User Id is for valid user... and maybe down the road validate that the User id is for the current user.)
https://guides.rubyonrails.org/active_record_validations.html#presence